test/e2e_federation/util.go

```diff
@@ -326,7 +326,7 @@ func cleanupServiceShardsAndProviderResources(namespace string, service *v1.Serv
 			zone := fedframework.GetZoneFromClusterName(name)
 			serviceLBName := cloudprovider.GetLoadBalancerName(cSvc)
 			framework.Logf("cleaning cloud provider resource for service %q in namespace %q, in cluster %q", service.Name, namespace, name)
-			framework.CleanupServiceResources(c.Clientset, serviceLBName, zone)
+			framework.CleanupServiceResources(c.Clientset, serviceLBName, "", zone)
```

Comment:
One more additional change comparing to the original commit. Assuming federation test does not run in regional upgrade test.

Reply:
> Assuming federation test does not run in regional upgrade test.

Oh never mind, in federation test `zone` is derived from cluster name. So we are good here.